carts/models.py

```diff
@@ -0,0 +1,50 @@
+from django.db import models
+
+
+class Product_Detail_Cart(models.Model):
```

Comment:
class 명에는 _ (언더바)를 사용하지 않습니다!
전체적으로 수정해주세요!

```suggestion
class ProductDetailCart(models.Model):
```

Reply:
> class 명에는 _ (언더바)를 사용하지 않습니다!
> 전체적으로 수정해주세요!

반영하였습니다.